### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by lhilmes-cb**
Updating product names and other language adjustments.

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [ ] Code follows the style guidelines of this project (PEP8, clean code).
- [ ] Linter has passed locally and any fixes were made for failures.
- [x] A self-review of the code has been done.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes (not tied to bugs/features)
- [ ] Other (please describe):

## What is the ticket or issue number?

- Ticket Number: N/A

- Issue Number: N/A

## Pull Request Description

Text changes to release notes only.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -85,45 +84,45 @@ Major Features
...
-    The new cbapi now provides a robust interface to the CB Response Live Response capability.
+    The new CBAPI now provides a robust interface to the Carbon Black EDR Live Response capability.
     Easily create Live Response sessions, initiate commands on remote hosts, and pull down data as
     necessary to make your Incident Response process much more efficient and automated.
 
-- **Consistent API for CB Response, Protection and Defense platforms**
-    We now support CB Response, Protection and Defense users in the same API layer. Even better,
-    the object model is the same for both; if you know one API you can easily transition to the other. cbapi
-    hides all the differences between the three REST APIs behind a single, consistent Python-like interface.
+- **Consistent API accross VMware Carbon Black platforms**
```
Comment (llyon-cb) [inline docs/index.rst:91]: typo here -- "accross" -> "across"

File: docs/index.rst [hunk 2]
```diff
@@ -235,10 +233,9 @@ Documentation (below) for detailed information on the objects and methods expose
 API Documentation
 -----------------
 
-Once you've taken a look at the User Guide, read through some of the
-`examples on GitHub <https://github.com/carbonblack/cbapi-python/tree/master/examples>`_,
-and maybe even written some code of your own, the API documentation can help you get the most out of cbapi by
-documenting all of the methods available to you.
+Once you have read the User Guide, youc an view `examples on GitHub <https://github.com/carbonblack/cbapi-python/tree/master/examples>`_
```
Comment (llyon-cb) [inline docs/index.rst:236]: typo here -- "youc an" -> "you can"

File: docs/index.rst (hunk 1 above)
Comment (lhilmes-cb) [reply docs/index.rst]: thanks for catching!

File: docs/index.rst (hunk 2 above)
Comment (lhilmes-cb) [reply docs/index.rst]: fixed.

**Pull request merged**